pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -5083,8 +5092,9 @@ func TestGetProcessMigrationTemplate(t *testing.T) {
 					Spec: api.KieAppSpec{
 						Environment: api.RhpamTrial,
 						CommonConfig: api.CommonConfig{
-							AdminUser:     "testuser",
-							AdminPassword: "testpassword",
+							AdminUser:              "testuser",
+							AdminPassword:          "testpassword",
+							SecretAdminCredentials: constants.KIE_ADMIN_CREDENTIALS_SECRET,
```

Comment:
Why there are changes for existing tests? We should keep all previous tests as they are and add new tests for secretAdminCredentielas. 
Please revert changes for existing defaults tests.

Reply:
ok, reverted